src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Card/CardPaymentsResource.cs

```diff
@@ -0,0 +1,34 @@
+﻿using SwedbankPay.Sdk.Common;
+using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    public class CardPaymentsResource : ResourceBase, ICardResource
+    {
+        public CardPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public async Task<ICardPaymentResponse> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)
+        {
+            if (id == null)
+                throw new ArgumentNullException(nameof(id));
+            Uri url = id.GetUrlWithQueryString(paymentExpand);
+
+            var cardPaymentResponseDto = await HttpClient.GetAsJsonAsync<CardPaymentResponseDto>(url);
+            return new CardPaymentResponse(cardPaymentResponseDto, HttpClient);
+        }
+
+
+        public async Task<ICardPaymentResponse> Create(CardPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None)
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)